### PR TITLE
keywords

### PR DESCRIPTION
**Pull request by mbostock**

### CODE REVIEW

File: docs/inputs/range.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+keywords: sliders
```
Comment (eagereyes) [inline docs/inputs/range.md:2]: Maybe slider instead?

File: docs/javascript/files.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+keywords: file, fileattachment, attachment
```
Comment (tophtucker) [inline docs/javascript/files.md:2]: just earlier today i was trying to get to this page typing blindly fast and landed on file input!! thank you 🙏

**Review by tophtucker — APPROVED** [review]
feels much better

File: docs/inputs/range.md (hunk 1 above)
Comment (mbostock) [reply docs/inputs/range.md:2]: It does prefix matching so this will match both.

**Pull request merged**